apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatView.kt

```diff
@@ -1497,6 +1504,108 @@ private fun TopEndFloatingButton(
   }
 }
 
+@Composable
+private fun FloatingDate(
+  modifier: Modifier,
+  listState: LazyListState,
+) {
+  var nearBottomIndex by remember { mutableStateOf(-1) }
+  var isNearBottom by remember { mutableStateOf(true) }
+  val lastVisibleItemDate = remember(listState.layoutInfo.visibleItemsInfo.lastIndex, listState.firstVisibleItemIndex, chatModel.chatItems) {
```

Comment:
try to remove everything in `remember(...)` to `remember {}` and see if it reduces recompositions and gives correct date at the same time

Reply:
maybe to keep chatModel.chatItems there if it will not work correctly without it